cypress.config.mjs

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    baseUrl: "https://tjommiboy.github.io/Workflow_CA",
```

Comment:
```suggestion
    baseUrl: "/",
```

You should be testing code in this branch, not code that has already been merged and deployed. Cypress runs it's own server.